packages/web/src/components/result/ReactiveList.js

```diff
@@ -661,6 +661,7 @@ class ReactiveList extends Component {
 			css={sortOptions}
 			className={getClassName(this.props.innerClass, 'sortOptions')}
 			name="sort-options"
+			aria-label="Sort options"
```

Comment:
@anik-ghosh-au7 Why do we have this change here?

Reply:
@bietkul this was added to avoid accessibility error as was indicated by Axe

Reply:
@bietkul removed